app/views/admin/merchants/index.html.erb

```diff
@@ -1,5 +1,5 @@
 <% @merchants.each do |merchant| %>
   <ul>
-    <li><%= merchant.name %></li>
+    <li><%= link_to "#{merchant.name}", "/admin/merchants/#{merchant.id}", method: :get %></li>
```

Comment:
I think the interpolation on the link name here is a bit superfluous since it's already erb.